sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusProcessorTest.java

```diff
@@ -442,39 +367,7 @@ public void testProcessorWithTracingEnabled(boolean isV2) throws InterruptedExce
 
     @Test
     @SuppressWarnings("unchecked")
-    public void testProcessorWithTracingEnabledAndNullMessage() throws InterruptedException {
```

Comment:
This test is not required anymore since v2 no longer uses `ServiceBusReceivedMessageContext` to funnel error. Which means the instrumentation in v2 processor will not get `ServiceBusReceivedMessageContext` with no message i.e. with error. Originally, this test was added after a regression where instrumentation tried to access null message in such `ServiceBusReceivedMessageContext`leading to NPE